ext_emconf.php

```diff
@@ -1,31 +1,31 @@
 <?php
 
 $EM_CONF[$_EXTKEY] = [
-    'title' => 'typo3_forum',
-    'description' => 'Forum extension',
-    'category' => 'plugin',
-    'author' => 'Mittwald CM Service',
-    'author_email' => 'support@mittwald.de',
-    'author_company' => 'Mittwald CM Service',
-    'dependencies' => 'cms,extbase,fluid,sr_feuser_register,static_info_tables',
-    'state' => 'stable',
-    'uploadfolder' => 0,
-    'createDirs' => 'typo3temp/typo3_forum,typo3temp/typo3_forum/gravatar',
-    'modify_tables' => 'fe_users',
-    'clearCacheOnLoad' => 0,
+    'title'            => 'typo3_forum',
+    'description'      => 'Forum extension',
+    'category'         => 'plugin',
+    'author'           => 'Mittwald CM Service',
+    'author_email'     => 'support@mittwald.de',
+    'author_company'   => 'Mittwald CM Service',
+    'dependencies'     => 'cms,extbase,fluid,sr_feuser_register,static_info_tables',
+    'state'            => 'stable',
+    'uploadfolder'     => true,
+    'createDirs'       => 'typo3temp/typo3_forum,typo3temp/typo3_forum/gravatar,uploads/tx_typo3forum/workflowstatus,uploads/tx_typo3forum/attachements',
+    'modify_tables'    => 'fe_users',
+    'clearCacheOnLoad' => false,
```

Comment:
Lining up thing vertically is fine with me for readability!
Why you removed it in the other commit in the annotations?
Give us the requested time for review.
Thanks!!!

Reply:
I just used auto format in PHPStorm for this one, maybe I didn't in the other one :-)

Reply:
don't mind ;-)